bin/cron/update_census_mapbox

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env ruby
+require_relative 'only_one'
+abort 'Script already running' unless only_one_running?(__FILE__)
+
+# The map on code.org/yourschool is run off of Mapbox.
+# This script updates the data on Mapbox by querying all rows from
+# census_summaries table for the appropriate school year,
+# creating maptiles with Mapbox's command line tool, Tippecanoe,
+# and uploading them to Mapbox's server.
+
+# After setting up authentication, provide the following CDO.* attributes:
+#   CDO.mapbox_upload_token: (the auth token for uploading tiles to Mapbox's servers)
+
+require_relative '../../dashboard/config/environment'
+require_relative '../../pegasus/helpers/properties'
+
+require_relative 'upload_to_mapbox'
+
+def make_location(lat, long)
+  return nil if lat.nil? || long.nil?
+
+  "#{format('%.6f', lat)}, #{format('%.6f', long)}"
+end
+
+def main
+  # Update summaries before updating the fusion table
+  # If any single teaches_cs category in a given year changes by more than
+  # 10% of the total number of schools then issue a Honeybadger notification.
```

Comment:
Do we still want this since we're redoing the different teaches_cs categories so there will probably be a lot of changes to this year?

Reply:
good question! no, this check seems like a relic from when engineering owned the algorithm. the RED team will be responsible for ensuring the integrity of this data now.